lib/algorithms/searches/linear_search.dart

```diff
@@ -0,0 +1,25 @@
+
+int linearSearch(List<int> list, int find) {
+  var index = list.length;
+
+  for(var i = 0; i < index; i++){
+    if(list[i] == find) {
+      return i;
+    }
+  }
+    return -1;
+}
+
+void main() {
+  final listTestCases = <List<int>>[
+    [],
+    [1],
+    [1, 5, -100, 50, 30032],
+    [1, 4, 2, 6, 30, 56, 46, 5, 24, 58, 3, 4444],
+  ];
+
+  for (var testCase in listTestCases) {
+    print(linearSearch(testCase, 5));
+  }
```

Comment:
I know I wrote this code as a quick test so we could see how your code worked today, so just for examples sake, this is just a "nit" as some call it (short for nit-pick). Lets pretend this was some debugging code you had and forgotten about when doing a quick and dirty test of your code or something; usually in Code Reviews (CR) you'll see someone maybe say something like the following:
> nit: probably some left over debug code that should get cleaned up before merge, but if not, could you explain why this is here? Might want to move this to some example file or modify and use in a test file.